modules/helm/Makefile.repo

```diff
@@ -43,6 +43,8 @@ helm\:repo\:add-current:
 	
 helm\:repo\:add:
 	@$(HELM) repo add kubernetes-charts http://storage.googleapis.com/kubernetes-charts
+	@$(HELM) repo add opsgoodness http://charts.opsgoodness.com
+	@$(HELM) repo add inclubator https://kubernetes-charts-incubator.storage.googleapis.com
```

Comment:
Typo `inclubator` should be `incubator`. But I suggest using official name `kubernetes-charts-incubator`